bundler/spec/dependabot/bundler/file_updater_spec.rb

```diff
@@ -878,6 +881,56 @@
           end
         end
 
+        context "when a git source is specified that multiple deps use for bundler v2", :bundler_v2_only do
+          let(:dependency_files) do
+            project_dependency_files("bundler2/git_source_with_multiple_deps")
+          end
+          let(:dependency) do
+            Dependabot::Dependency.new(
+              name: "elasticsearch-dsl",
+              version: "86a36ec0db704b2a62dd4d5fe9edf887625b1826",
+              previous_version: "43f48b229a975b77c5339644d512c88389fefafa",
+              requirements: requirements,
+              previous_requirements: previous_requirements,
+              package_manager: "bundler"
+            )
+          end
+          let(:requirements) { previous_requirements }
+          let(:previous_requirements) do
+            [{
+              file: "Gemfile",
+              requirement: ">= 0",
+              groups: [],
+              source: {
+                type: "git",
+                url: "https://github.com/elastic/elasticsearch-ruby.git",
+                branch: "5.x",
+                ref: "5.x"
+              }
+            }]
+          end
+
+          it "updates the dependency's revision" do
```

Comment:
Is this true? 😅 seems like the test is checking if we keep the same revision as the old lockfile? 🤔 

Looks like the requirements are not being updated at all so not clear what's being tested. What happens if we do end up updating the requirement? Do we need to unlock the other gems also?

Reply:
This was mostly just adapted from the v1 version, but I think what this is supposed to test is that we keep the same remote after the update, but that it does not have the same revision:

```
            expect(new_remote_line).to eq(original_remote_line)
            expect(new_revision_line).to_not eq(original_revision_line)
```

This seems right to me?

Reply:
Oh yes you're right, totally missed the `to_not eq` bit! 👍

Reply:
Let's chance it to `not_to` 👍